recipes/libcurl/all/conanfile.py

```diff
@@ -373,77 +385,101 @@ def _get_configure_command_args(self):
+            tc.configure_args.append("--disable-proxy")
 
         if not self.options.with_rtsp:
-            params.append("--disable-rtsp")
+            tc.configure_args.append("--disable-rtsp")
 
         if not self.options.with_crypto_auth:
-            params.append("--disable-crypto-auth") # also disables NTLM in versions of curl prior to 7.78.0
+            tc.configure_args.append("--disable-crypto-auth") # also disables NTLM in versions of curl prior to 7.78.0
 
         # ntlm will default to enabled if any SSL options are enabled
         if not self.options.with_ntlm:
-            if tools.Version(self.version) <= "7.77.0":
-                params.append("--disable-crypto-auth")
+            if Version(self.version) <= "7.77.0":
+                tc.configure_args.append("--disable-crypto-auth")
             else:
-                params.append("--disable-ntlm")
+                tc.configure_args.append("--disable-ntlm")
 
         if not self.options.with_ntlm_wb:
-            params.append("--disable-ntlm-wb")
+            tc.configure_args.append("--disable-ntlm-wb")
 
         if self.options.with_ca_bundle == False:
-            params.append("--without-ca-bundle")
+            tc.configure_args.append("--without-ca-bundle")
         elif self.options.with_ca_bundle:
-            params.append("--with-ca-bundle=" + str(self.options.with_ca_bundle))
+            tc.configure_args.append("--with-ca-bundle=" + str(self.options.with_ca_bundle))
 
         if self.options.with_ca_path == False:
-            params.append('--without-ca-path')
+            tc.configure_args.append('--without-ca-path')
         elif self.options.with_ca_path:
-            params.append("--with-ca-path=" + str(self.options.with_ca_path))
+            tc.configure_args.append("--with-ca-path=" + str(self.options.with_ca_path))
 
         # Cross building flags
         if cross_building(self):
             if self.settings.os == "Linux" and "arm" in self.settings.arch:
-                params.append("--host=%s" % self._get_linux_arm_host())
+                tc.configure_args.append("--host=%s".format(self._get_linux_arm_host()))
             elif self.settings.os == "iOS":
-                params.append("--enable-threaded-resolver")
-                params.append("--disable-verbose")
+                tc.configure_args.append("--enable-threaded-resolver")
+                tc.configure_args.append("--disable-verbose")
             elif self.settings.os == "Android":
                 pass # this just works, conan is great!
 
-        return params
+        # tweaks for mingw
+        if self._is_mingw:
+            rcflags = "-O COFF"
+            if self.settings.arch == "x86":
+                rcflags += " --target=pe-i386"
+            else:
+                rcflags += " --target=pe-x86-64"
+            env = tc.environment()
+            env.define("RCFLAGS", rcflags)
+
+            tc.extra_defines.append("_AMD64_")
+
```

Comment:
I don't understand this error: `extra_defines` is documented here: https://docs.conan.io/en/latest/reference/conanfile/tools/gnu/autotoolstoolchain.html#conan-tools-gnu-autotools-toolchain

Reply:
Ok it seems that it was related to the conan version. Seems to be gone since conan 1.51.3 is used for the CI